notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,949 @@
+{
```

Comment:
<p>I wonder whether one model per team member is a bit tough for them? we could say something like at least 3 models per team, one of which may be a random forest. we might also want to point them to some sklearn links, e.g. linear regression, extratrees, and <a href="https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html" target="_blank">https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>My thinking was that this way there would be one part of the notebook that would be unique to the team members. </p>

Reply:
<p>i see, so how about we </p><p> </p><p>a) point them to suitable models to play with (i.e. enough choices for each group, probably 4 models max)</p><p>b) rephrase the test to "train at least one model per team member. You can only have one Random Forest model per team, so you may want to investigate the following alternatives for regression: ..."</p>